<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>depozit_calc</class>
 <widget class="QDialog" name="depozit_calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Depozit_Calc</string>
  </property>
  <widget class="QLabel" name="label_sum">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма вклада:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>0</y>
     <width>16</width>
     <height>401</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_sum_depozit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>18</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000000000001.000000000000000</double>
   </property>
   <property name="value">
    <double>10000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_term">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок размещения:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_term">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>58</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000000000</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_term">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>58</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>В месяцах</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>В годах</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_Interest_rate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Процентная ставка:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_interest_rate">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>96</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>100000000000000004188152556421145795899143386664033828314342771180699648.000000000000000</double>
   </property>
   <property name="value">
    <double>2.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_Interest_rate_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tax_rate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Налоговая ставка:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tax_rate_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_tax_rate">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>136</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100000000000000004188152556421145795899143386664033828314342771180699648.000000000000000</double>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_capitalization">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Капитализация процентов</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_frequency">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>185</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Переодичность выплат:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_frequency">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Раз в месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в год</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_replenishment">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>262</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ежемесячное пополнение:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_replenishment">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000000000001.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_replenishment_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>302</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ежемесячное снятие:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_withdrawal">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000000000001.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Расчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Interest">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>150</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Начисленные проценты:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Interest_total">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>150</y>
     <width>421</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tax">
   <property name="geometry">
    <rect>
     <x>482</x>
     <y>180</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма налога:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tax_total">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>180</y>
     <width>421</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_end_balance">
   <property name="geometry">
    <rect>
     <x>412</x>
     <y>210</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Итоговая сумма вклада:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_balance">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>210</y>
     <width>421</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>503</x>
     <y>223</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(к концу срока)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
